sdk/monitor/azure-monitor-query/azure/monitor/query/_log_query_client.py

```diff
@@ -131,7 +131,7 @@ def query(self, workspace_id, query, duration=None, **kwargs):
             process_error(e)
 
     def batch_query(self, queries, **kwargs):
```

Comment:
Shouldn't it return a pageable object?

Reply:
no - service doesn't support paging

Reply:
I think our guideline is even service does not support page, we should wrap the return into pageable results?
@annatisch

Reply:
we talked to the architects and they were fine in this case with not supporting both paging and LRO
as service is not planning to ever support it on these endpoints

Reply:
i have created this issue for september milestone to follow up again https://github.com/Azure/azure-sdk-for-python/issues/19942